src/components/NameInput/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { InputHTMLAttributes, ReactNode } from 'react';
+import styles from './styles.module.css';
+import { IconType } from 'react-icons';
+type NameInput = InputHTMLAttributes<HTMLInputElement> & {
+    title?: string
+    icon?: IconType
+}
+export function NameInput({title,icon: Icon,...props}: NameInput) {
+    return (
+        <>
+        <div className={styles.content}>
+            <span className={styles.title}>{title}</span>
+            <input className={styles.input} {...props}/>
+            {Icon && (
+            <div className={styles.iconContent}>
+                <span className={styles.icon}>
+                    <Icon/>
+                </span>
+            </div>)}
+        </div>
```

Comment:
A gente discutiu que teremos um componente base `<Input />` que seria usado para todos os outros input. Isso corresponde a esta task no trello: https://trello.com/c/9jhkwYC1/6-input 
Então essa task fica bloqueada até a task do Input for concluida.